### PR TITLE
Feature/register

### PR DESCRIPTION
**Pull request by Salvarez0212**
## Summary
Added inputs for blood type, gender, and country that will be display in the profile page

image

### CODE REVIEW

File: src/pages/register/components/SelecBlood.register.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    >
+      <option value="">Select...</option>
+      <option value="A+">A+</option>
+      <option value="A-">A-</option>
+      <option value="B+">B+</option>
+      <option value="B-">B-</option>
+      <option value="O+">O+</option>
+      <option value="O-">O-</option>
+      <option value="AB+">AB+</option>
+      <option value="AB-"></option>
```
Comment (ht1204) [inline src/pages/register/components/SelecBlood.register.jsx:19]: This option item should be:
```html
<option value="AB-">AB-</option>
```

File: src/pages/register/components/SelectCountry.register.jsx [hunk 2]
```diff
@@ -0,0 +1,276 @@
+export const SelecCountry = ({ handleChange }) => {
```
Comment (ht1204) [inline src/pages/register/components/SelectCountry.register.jsx:1]: This should be named:
```js
export const SelectCountry = ({ handleChange }) => {
```

File: src/pages/register/components/SelectCountry.register.jsx [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+      <option value="Latvia">Latvia</option>
+      <option value="Lebanon">Lebanon</option>
+      <option value="Lesotho">Lesotho</option>
+      <option value="Liberia">Liberia</option>
+      <option value="Libyan Arab Jamahiriya">Libyan Arab Jamahiriya</option>
+      <option value="Liechtenstein">Liechtenstein</option>
+      <option value="Lithuania">Lithuania</option>
+      <option value="Luxembourg">Luxembourg</option>
+      <option value="Macao">Macao</option>
+      <option value="Macedonia, the Former Yugoslav Republic of">
```
Comment (ht1204) [inline src/pages/register/components/SelectCountry.register.jsx:148]: Currently the suggested name is Northern Macedonia.

File: src/pages/register/components/SelectCountry.register.jsx [hunk 4]
```diff
@@ -0,0 +1,276 @@
...
+      <option value="Venezuela">Venezuela</option>
+      <option value="Viet Nam">Viet Nam</option>
+      <option value="Virgin Islands, British">Virgin Islands, British</option>
+      <option value="Virgin Islands, U.s.">Virgin Islands, U.s.</option>
+      <option value="Wallis and Futuna">Wallis and Futuna</option>
+      <option value="Western Sahara">Western Sahara</option>
+      <option value="Yemen">Yemen</option>
+      <option value="Zambia">Zambia</option>
+      <option value="Zimbabwe">Zimbabwe</option>
+    </select>
```
Comment (ht1204) [inline src/pages/register/components/SelectCountry.register.jsx:274]: I consider this list of countries should come from an object or an array and create a dynamic rendering with a loop, I suggest work on this enhancement.

File: src/pages/register/components/SelecBlood.register.jsx (hunk 1 above)
Comment (ht1204) [reply src/pages/register/components/SelecBlood.register.jsx:19]: I suggest an enhancement as I mention in list of countries, create an object or array an create a dynamic rendering with a loop, should be better an easier to mantain.

File: src/pages/register/Register.page.jsx [hunk 5]
```diff
@@ -104,6 +109,32 @@ export const Register = () => {
...
+              onChange={(event) => handleChange(event)}
+            >
+              <option value="male">Male</option>
+              <option value="female">Female</option>
+              <option value="non-binary">Non binary</option>
+            </select>
+          </div>
+
+          <div className="country__container input__container">
+            <label htmlFor="country">Selec your country:</label>
```
Comment (ht1204) [inline src/pages/register/Register.page.jsx:129]: Watch this:
```html
<label htmlFor="country">Select your country:</label>
```

File: src/pages/register/Register.page.jsx [hunk 6]
```diff
@@ -23,6 +27,7 @@ export const Register = () => {
     const { name, type, value, checked } = event.target;
     setData({ ...data, [name]: type === 'checkbox' ? checked : value });
   };
+  console.log(data);
```
Comment (ht1204) [inline src/pages/register/Register.page.jsx:30]: What is this console.log doing here?

**Review by ht1204 — CHANGES_REQUESTED** [review]
Consider and address according to the comments, please and thanks in advance.

**Comment by Salvarez0212** [comment]
I have completed all the suggestions you asked for, I stay tuned if another correction is needed!

**Review by JeanVittory — DISMISSED** [review]

**Review by ht1204 — APPROVED** [review]

**Pull request merged**